registry/data-models/models.py

```diff
@@ -0,0 +1,171 @@
+data models and backend API.
+Diagram of the data models:
+                                  +-----------------+
+                                  |     Project     |
+                                  +-----------------+
+                                          |  
+                                          |
+                                          | 
+                            +------------------------------+    
+                            |                              |      
+                            | 1:n                          | 1:n         
+                  +------------------+           +------------------+
+      +---------- |   FeatureName    |           |     Anchor       |
+      |      1:n  +------------------+           +------------------+
+      |                 ｜                             ｜          |
+      |                 ｜ 1:n                         ｜1:n       |
+      |                 +------------------------------+          |
+      |                               ｜                           |
+      |                               ｜1:n                       \|/ has
+      |   +--------------+    +---------------+                 +----------------+
+      |   |Transformation|----|    Feature    |                 |   DataSource   |            
+      |   +--------------+ has+---------------+                 +----------------+
+      |                              /|\                         /|\           |extends
+      |                               |                           |            |
+      |                               |                           |            |
+      |                               |                           |            |
+      |          +--------------------------------------+         |           \|/
+      |          |                                      |         |        +----------+
+      |          |extends                               |extends  |has     |  Source  |
+      |    +----------------+                      +-----------------+     +----------+
+      |    | DerivedFeature |                      |  AnchorFeature  |          /|\
+      |    +----------------+                      +-----------------+           |
+      |            |                                                             |
+      |            |                                                             |extends
+      |            |                                                       +------------------+
+      |            +-------------------------------------------------------|  FeaturesSource  |
+      |                                                                has +------------------+
+      |                                                                             |
+      +-----------------------------------------------------------------------------|                                 
+"""
+
+
+class FeatureId(BaseModel):
```

Comment:
Can we provide a brief context on these id models?

Reply:
I added comment, please review again.